### PR TITLE
Labyrinth - Maria Pogosjan & Fethullah Dincer 

### PR DESCRIPTION
**Pull request by MariaPogosjan**
Link to my deployed project:
https://meze-game.netlify.app/

My reflections on how this week's project turned out:

Things I'd like to have clarified or explained in more detail:
We did not manage to find a unique key from the API when we wore mapping. How should we solve it?

### CODE REVIEW

**Review by klimenko3007 — CHANGES_REQUESTED** [review]
Great job, guys! Your code looks very clean and easy to read! No forgotten console.logs anywhere :D  There are couple of issues though when it comes to styling. It does not always scale well on small mobile devices. We added a couple of suggestions on how to fix it.

File: code/src/components/Description.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+              <button 
+                className='nes-btn is-warning'
+                type='button'
+                onClick={() => {dispatch(generateMove(userName, action.direction))}}  
+              >
+                {action.direction}
+              </button>
+            </div>
+          </div>
+        ))}
```
Comment (KarolCislowski) [inline code/src/components/Description.js:34]: map has an extra parameter- index. 

```jsx
  arr.map((action, index) => (
    <Component key={`key-${index}`}
  )
```

File: code/src/components/WelcomePage.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+            onChange={e => setInputValue(e.target.value)}
+          />
+        </div>
+        <button 
+          onClick={onNameSet} 
+          type='button' 
+          className='nes-btn is-warning'
+        >
+          ENTER
+        </button>
```
Comment (KarolCislowski) [inline code/src/components/WelcomePage.js:43]: It is not nice that you forgot about the form tag. It striped your input from submission on enter, what is the most used sending the form method 😭

File: code/src/components/ActionHistory.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+const ActionHistory = () => {
+  const history = useSelector(store => store.games.history)
+
+  return (
+    <>
+      <div className='nes-container is-dark with-title'>
+        <p>Your actions:</p>
+        {history.map((option) => (
+          <div className='history-container'>
+            <p className='action-text'>{option}</p>
```
Comment (KarolCislowski) [inline code/src/components/ActionHistory.js:13]: It would be nice to separate every word with space 🤓

**Review by KarolCislowski — APPROVED** [review]
Awesome app 😺

**Pull request closed without merging**